src/modm/driver/radio/sx128x.hpp

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) 2023, Lasse Alexander Jensen
+ * Copyright (c) 2023, Rasmus Kleist Hørlyck Sørensen
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_SX128X_HPP
+#define MODM_SX128X_HPP
+
+#include "sx128x_definitions.hpp"
+#include "sx128x_transport.hpp"
+
+#include <modm/processing/timer.hpp>
+
+namespace modm
+{
+
+/**
+ * @tparam	Transport   Transpot Layer for SX128x
+ *
+ * @ingroup modm_driver_sx128x
+ * @author	Lasse Alexander Jensen
+ * @author  Rasmus Kleist Hørlyck Sørensen
+ */
+template< class Transport, class Reset, class Busy >
+class Sx128x : public sx128x, public Transport
+{
+    using Command = Transport::Command;
+
+public:
+    Sx128x() = default;
+
+    /// Determine if radio is busy
+    inline bool
```

Comment:
```suggestion
    bool
```
`inline` is not needed here because the class is a template.